src/app/users/view-user/view-user.component.scss

```
@@ -0,0 +1,10 @@
+.container {
+  max-width: 37rem;
+
+  .content {
+    div {
+      margin: 1rem 0;
+      word-wrap: break-word;
```

```suggestion
      word-wrap: break-word;
      
      span {
        display: block;
      }
```
To ensure every `role` is displayed in a separate line.